packages/react-native-session-replay/android/src/main/kotlin/com/datadog/sessionreplay/reactnative/DdSessionReplayImplementation.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.sessionreplay.reactnative
+
+import com.facebook.react.bridge.Promise
+
+/**
+ * The entry point to use Datadog's Session Replay feature.
+ */
+class DdSessionReplayImplementation() {
+    /**
+     * Enable session replay and start recording session.
+     * @param replaySampleRate The sample rate applied for session replay.
+     * @param defaultPrivacyLevel The privacy level used for replay.
+     */
+    fun enable(replaySampleRate: Double, defaultPrivacyLevel: String, promise: Promise) {
```

Comment:
seems like this method doesn't actually enable anything? is it on purpose?

Reply:
In this PR I want to make sure that all native packages work (i.e. the app does not crash when they're called) without any actual logic in them (it's mostly "plumbing" work).
In the next PR I'll enable SR in this method.